pkgs/top-level/all-packages.nix

```diff
@@ -8569,13 +8588,15 @@ with pkgs;
       limesuite
       soapyairspy
       soapyaudio
-      soapybladerf
       soapyhackrf
-      soapyplutosdr
       soapyremote
       soapyrtlsdr
+    ]
+    ++ (lib.optionals stdenv.hostPlatform.isLinux [
+      soapybladerf
       soapyuhd
-    ];
+      soapyplutosdr
```

Comment:
soapyplutosdr builds on darwin, is it really need to exclude?

Reply:
Thanks. Somehow it was also not building before. Fixed.